ironfish-cli/src/commands/chain/export.ts

```diff
@@ -41,6 +40,17 @@ export default class Export extends IronfishCommand {
   async start(): Promise<void> {
     const { flags, args } = await this.parse(Export)
 
+    // Regular expression that accepts only english numbers
+    const regex = new RegExp('[^0-9]')
```

Comment:
We have very few regexes in our code - they are brittle and generally should be avoided in production code when possible.

You're checking a string for non-numeric value in typescript - there are good ways to do that without invoking regexes, such as `isNaN(Number(args.start))`

Reply:
Thanks @lwisne . I will work on this

Reply:
@lwisne I updated the code, as well as updated the output